data/ecosystems/e/easidao.toml

```diff
@@ -3,59 +3,17 @@ title = "EasiDAO"
 
 sub_ecosystems = []
 
-github_organizations = ["https://github.com/EaseDeFi"]
+github_organizations = ["https://github.com/Hedera-Varmeta"]
 
 # Repositories
 [[repo]]
-url = "https://github.com/EaseDeFi/addon-whitelist"
+url = "https://github.com/Hedera-Varmeta/Easi-DAO-smartcontract"
 
 [[repo]]
-url = "https://github.com/EaseDeFi/arCore"
+url = "https://github.com/Hedera-Varmeta/Easi-DAO-Backend"
 
 [[repo]]
-url = "https://github.com/EaseDeFi/armor-governance"
+url = "https://github.com/Hedera-Varmeta/Easi-DAO-Frontend"
 
 [[repo]]
-url = "https://github.com/EaseDeFi/arNFT"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/arNXM"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/arShield"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/assets"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/Audits"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/DefiLlama-Adapters"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/ease-rca"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/ease-zapper-integration"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/gvToken"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/inedible-assets"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/smart-contracts"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/snapshot-spaces"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/synthetix-faq-bot"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/yield-server"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/yinsure-finance"
```

Comment:
why are you proposing to remove this repos?

Reply:
I already updated, thank you